shared/chat/conversation/container.js

```diff
@@ -41,6 +45,10 @@ class ConversationContainer extends Component<void, Props, State> {
     this.setState({sidePanelOpen: !this.state.sidePanelOpen})
   }
 
+  _onTriggerScrollDown = () => {
+    this.setState({listScrollDownCounter: this.state.listScrollDownCounter + 1})
```

Comment:
:sheep: Suggestion: a way to make this feel less arbitrary would be to have the store record the id of the latest posted message, and pass it down the list as a prop. Then you can trigger the scroll locking behavior as a result of a new message being sent.